uiowa/uiowa
# Feature: banner pre-title

Resolves https://github.com/uiowa/uiowa/issues/4713. 

# How to test

- Review files changed
- `blt frontend`
- `blt ds --site=sandbox.uiowa.edu`
- Add/edit a banner block and verify that:
  -  The pre-title field works. 
  - Styles match the heading style, except when the 'Highlight' version is selected.  When this style is selected the pre-title will appear with the same font style without the highlight style. 

## Story 

As a site owner, I can use a newly added pre-title field as a part of the banner block that appears above the banner title to a new or existing block.
